docs/api-docs/cmds.md

```diff
@@ -1,206 +1,19 @@
-      // Returning loops from a reducer is optional by branch. The store knows
-      // how to examine results and compose cmds into a separate effect tree
-      // from the state tree.
-      return { ...state, second: true };
-  }
-}
-```
-
-## `liftState(state): [any, Cmd]`
-
-* `state: any` &ndash; an object which may be the state of the redux store, or
-  an existing `[any, Cmd]` pair created by `loop()`.
-
-#### Notes
-
-Automatically converts objects to `loop()` results. If the value was created
-with `loop()`, then the function behaves as an identity. Otherwise, it is lifted
-into a `[any, Cmd]` pair where the effect is `Cmd.none()`. Useful for
-forcing reducers to always return a `loop()` result, even if they shortcut to
-just the model internally.
-
-#### Example
-
-```js
-function reducer(state, action) {
-  switch(action.type) {
-    case 'LOAD_START':
-      return loop(
-        { ...state, isLoading: true },
-        Cmd.run(apiFetch, {
-           successActionCreator: resolveActionCreator,
-           failActionCreator: rejectActionCreator,
-           args: [action.payload.id]
-        })
-    case 'LOAD_COMPLETE':
-      return {
-        ...state,
-        isLoading: false,
-        result: action.payload,
-      };
-    default:
-      return state;
-  }
-}
-
-// This guarantees that the return value of the reducer will be a loop result,
-// regardless of if it was set as such in the reducer implementation. This makes
-// it much easier to manually compose reducers without cluttering reducer
-// implementations with `loop(state, Cmd.none())`.
-export default compose(reducer, liftState);
-```
-
-
-## `getModel(loop): any`
-
-* `loop: any` &ndash; any object.
-* returns the model component of the array if the input is a `[any, Cmd]`
-  pair, otherwise returns the input object.
-
-#### Notes
-
-`getModel` lets you extract just the model component of an array returned by
-`loop`. It's useful in testing if you need to extract out the model component
-to do custom comparisons like `Immutable.is()`.
-
-
-## `getCmd(loop): Cmd | null`
-
-* `loop: any` &ndash; any object.
-* returns the cmd component of the array if the input is a `[any, cCmd]`
-  pair, otherwise returns `null`.
-
-#### Notes
-
-`getCmd` lets you extract just the cmd component of an array returned by
-`loop`. It's useful in testing if you need to separate the model and cmd and
-test them separately.
-
-
-## `isLoop(object): boolean`
- 
- * `object: any` &ndash; any object.
- * returns whether the given object was created with the `loop` function.
- 
- #### Notes
- 
- `isLoop` lets you determine whether an object returned by a reducer includes an
- cmd. This function is useful for writing custom higher-order functionality on
- top of redux-loop's API, or for just writing your own combineReducers.
-
-
-
-## `Cmds`
-
-#### Notes
+# `Cmds`
 
 The `Cmd` object provides access to all of the functions you'll need to
```

Comment:
I know you didn't write this part but I think the wording is a bit confusing. How about

The `Cmd` object provides access to all of the helpers you need to create cmd. A cmd is a plain object describing a side effect that can happen after your reducer runs. It is executed by the enhanced redux store outside of the reducer, which leaves the reducer pure and testable.

Reply:
I think this can be confusing because we have objects that are generated by functions like `Cmd.none()`, `Cmd.action()`, `Cmd.list()`, etc. It seems to me that those object which are generated by those functions are really what we should refer to as "command objects". We have a _module_ called `Cmd` which provides a bunch of convenience functions for generating command objects. Therefore I think it might make more sense to say something more like this:

> The `Cmd` module provides access to all of the functions you'll need to
create and represent different kinds of cmd objects to redux-loop's
cmd-processor. Every cmd object is simply a plain JavaScript object that describes a side-effect that can happen after your reducer runs. This cmd object is then executed by the enhanced redux store outside of the reducer, which leaves the reducer pure and testable.

Reply:
sounds good to me!